src/System.IO.FileSystem.Watcher/src/Interop/Interop.Linux.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+partial class Interop
+{
+    const string LIBC = "libc";
```

Comment:
This file should be renamed and restructured related to #531.  
- It should be in src\Common\src\Interop\Linux\libc\Interop.inotify.cs
- The LIBC constant should be removed.  
- "partial class Interop" should be changed to "internal static partial class Interop"
- Its contents should all be moved to a nested type "internal static partial class libc".